controllers/mirroring/mirroring_controller.go

```diff
@@ -31,10 +28,12 @@ import (
 
 	"github.com/go-logr/logr"
 	rookCephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	"golang.org/x/exp/maps"
```

Comment:
check if you can remove this, all exp of maps is already in stdlib afair.

Reply:
Tried removing this and it was throwing error